src/chainparams.cpp

```diff
@@ -197,9 +197,11 @@ class CMainParams : public CChainParams
                 // August 1st 2017 CASH fork (UAHF)
                 478559, uint256S("0x000000000000000000651ef99cb9fcbe0dadde1d424bd9f15ff20136191a5eec"))(
                 // November 13th 2017 new DAA fork
-                504031, uint256S("0x0000000000000000011ebf65b60d0a3de80b8175be709d653b4c1a1beeb6ab9c")),
-            1483472411, // * UNIX timestamp of last checkpoint block
-            184495391, // * total number of transactions between genesis and last checkpoint
+                504031, uint256S("0x0000000000000000011ebf65b60d0a3de80b8175be709d653b4c1a1beeb6ab9c"))(
+                // May 15th 2018 re-enable op_codes and 32 MB max block size
+                530359, uint256S("0x0000000000000000011ada8bd08f46074f44a8f155396f43e38acf9501c49103")),
+            1526410186, // * UNIX timestamp of last checkpoint block
+            249416375, // * total number of transactions between genesis and last checkpoint
```

Comment:
i think the total num transactions is a little high...this is what i get from txoutsetinfo


06:07:29
￼
{
  "height": 531175,
  "bestblock": "00000000000000000172f0cbdead4312dc4e0d42dcec3ea87b8f996d1588ece5",
  "transactions": 17862068,
  "txouts": 39837658,
  "hash_serialized_2": "c112904955a45b764aa5dbc18469b5d7dfe474da1ea2648fa454c4f8ff381487",
  "disk_size": 2241613078,
  "total_amount": 17139535.94435441
}